app/views/password_resets/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<% provide :title, t(".title") %>
+<h1><%= t ".header" %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for @user, url: password_reset_path( params[:id]) do |f| %>
```

Comment:
Tại sao form for ở đây những vẫn phải dùng thêm url em

Reply:
Ta có một số routes sử dụng password_reset_path như password_reset_path, new_password_reset_path...Khi tạo action là new thì sẽ là new_password_reset, khi  update sẽ là password_reset_path. Vì vậy dùng thêm url để xác định rõ ràng path hay đường dẫn tuyệt đối

Reply:
Ý anh hỏi ở đây là form_for nó sẽ tự sinh ra url rồi đúng không như sao ở đây vẫn cần thêm url nữa á

Reply:
url mà form_for tự sinh ở đây sẽ là url được xác định từ lớp đối tượng, trong trường hợp này là User. Nhưng URL phía sau mình đang sử dụng là password_reset không được xác định từ lớp User, vì vậy cần thêm URL vào để định nghĩa chính xác.